### PR TITLE
Add concrete acceptance steps to README

### PR DESCRIPTION
**Pull request by goldfirere**
In an attempt to share @nomeata's burden as secretary, I'd like to start merging my own acceptances. I believe I have outlined the concrete steps to take in this PR, but I would love confirmation.

For the rendered view, scroll up just a bit before [this heading](https://github.com/goldfirere/ghc-proposals/tree/concrete-acceptance#review-criteria).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -395,6 +395,32 @@ is a polite ping/enquiry.
...
+            to "was discussed".
+
+      2. Merge the PR branch into master, and push.
+
+      3. Update the PR description:
+
+         a. Update the "Rendered" link to point to the permanent home
+         of the accepted proposal.
+
+         b. Add the text "The proposal has been accepted; the following discussion is mostly of historic interest."
```
Comment (nomeata) [inline README.rst:417]: Actually, I leave the description alone and add

```
The [proposal](https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/… has been accepted; the following discussion is mostly of historic interest.

---
```
above. Nobody has to look below the `--`. More uniform.

File: README.rst [hunk 2]
```diff
@@ -395,6 +395,32 @@ is a polite ping/enquiry.
...
+
+      4. Set the PR to have the "Accepted" label.
+
+      5. Comment on the PR that the proposal was accepted.
+
+      6. Close the PR if GitHub has not detected the merge.
```
Comment (nomeata) [inline README.rst:423]:
```suggestion
      6. Close the PR if GitHub has not detected the merge.

      7. Announce on the committee mailing list

```

Comment (nomeata) [reply README.rst:423]: And also send a tweet if you want … :-)

File: README.rst (hunk 1 above)
Comment (nomeata) [reply README.rst:417]: Also, remove `(under review)` from the PR title if it is there

**Comment by goldfirere** [comment]
Having taken all of @nomeata's suggestions, and noting that he is the final word on these matters, I am merging now.

**Pull request merged**